### PR TITLE
created README.md file and edited in markdown

### PR DESCRIPTION
**Pull request by Vee-keme**

### CODE REVIEW

**Comment by Vee-keme** [comment]
@owans @Roxie-32

File: TMP2022/technical-writing-track/precious-tw-funke/README.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+# Kuda Bank User Guide For Android Users
+
+Nigeria's first mobile-only bank, [Kuda Bank](https://kuda.com/), was granted a license by the Central Bank of Nigeria (CBN) in 2019. Consequently, Kuda is only available online and has no physical locations. Babs Ogundeyi and Musty Mustapha formed Kuda Bank. This guide illustrates getting started with the Kuda bank app.
+
+We'll go over creating a Kuda account and using the app's fundamental features. Download your app on the [Google Playstore](https://play.google.com/store/apps/details?id=com.kudabank.app&hl=en&gl=US) store for android devices and Let's get started.
```
Comment (owans) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]: Create a table of content to link inner pages or link the pages in the introduction. For e.g, [Creating a Kuda Account](#create-account)

File: TMP2022/technical-writing-track/precious-tw-funke/README.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+- Virtual Kuda Visa Card
+  Kuda provides you with a Virtual Naira Visa card which allows you make payments online on anywhere naira visa cards are accepted.
+
+One of the perks of using the kuda card is the control you have. Say, for example, your physical card gets stolen you can easily [block your card](https://help.kuda.com/en/articles/3365550-block-your-kuda-card) on the app.
+
+# GETTING HELP
+
+If ever you need help you can send an email to Kuda [here](help@kuda.com) or call 01 633 5832 between 8:00 am and 5:00 pm on weekdays.
+
+[def]: ./pictures/1.png
```
Comment (owans) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]: What does this link to?

File: TMP2022/technical-writing-track/precious-tw-funke/README.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+_N/B: your transaction pin and your passcode are not the same thing. The former is for performing transactions while the latter is for accessing your kuda dashboard._
+
+## More About You
+
+This section focuses on how you intend to use kuda. You could, for example, use kuda for everyday transactions such as savings, debit card, and so on. It also asks questions such as how much you earn and what type of work you do in order to provide you with a wholesome experience when using Kuda Bank.
+
+When the more about survey is completed, you can access your dashboard and start transacting.
+
+# EXISTING USERS
```
Comment (owans) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]: Check to use the proper heading here. Is this supposed to be a heading One or Two?

File: TMP2022/technical-writing-track/precious-tw-funke/README.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
+# Kuda Bank User Guide For Android Users
+
+Nigeria's first mobile-only bank, [Kuda Bank](https://kuda.com/), was granted a license by the Central Bank of Nigeria (CBN) in 2019. Consequently, Kuda is only available online and has no physical locations. Babs Ogundeyi and Musty Mustapha formed Kuda Bank. This guide illustrates getting started with the Kuda bank app.
```
Comment (owans) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]:
```suggestion
Nigeria's first mobile-only bank, [Kuda Bank](https://kuda.com/), was granted a license by the Central Bank of Nigeria (CBN) in 2019. Consequently, Kuda is only available online and has no physical locations. Babs Ogundeyi and Musty Mustapha formed Kuda Bank. This guide illustrates how to get started with the Kuda Bank App.
```

File: TMP2022/technical-writing-track/precious-tw-funke/README.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+We'll go over creating a Kuda account and using the app's fundamental features. Download your app on the [Google Playstore](https://play.google.com/store/apps/details?id=com.kudabank.app&hl=en&gl=US) store for android devices and Let's get started.
+
+# CREATE ACCOUNT
+
+You have control over your finances, spending, and savings when you have a Kuda account. You may accomplish all of this with the assistance of Kuda, commonly known as the bank of the free, with almost no charges. Let's look at creating a Kuda account.
+
+## Open An Account.
```
Comment (owans) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]: Break these down into steps and add images for each step

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 4 above)
Comment (Roxie-32) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]: Babs Ogundeyi and Musty Mustapha founded Kuda Bank.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+Kuda requests for your BVN so they can be sure no one is pretending to be you.
+When you confirm your BVN, Kuda matches your details with the information on the national BVN database owned by the Central Bank of Nigeria. Other of your account(s) will not be accessed with your BVN.
+Secondly, Kuda uses Artificial intelligence to record your face.
+
+## Secure Your Account
+
+Kuda will request that you create a transaction pin. Your transaction pin is a 4-digit pin that allows you perform monetary transactions across kuda.
+
+_N/B: your transaction pin and your passcode are not the same thing. The former is for performing transactions while the latter is for accessing your kuda dashboard._
```
Comment (Roxie-32) [inline TMP2022/technical-writing-track/precious-tw-funke/README.md]: Can be a blockqoute to draw emphasis, check how to use blockquote in markdown

**Comment by Roxie-32** [comment]
@Vee-keme you may want to work on your headings, it's a bit scattered.  Your subheadings of heading 3 should be under heading 2. If you are talking about an entirely new concept move back to heading 2. If you use heading 3 we assume its still a sub heading of the previous heading 2.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 2 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: got rid of it

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 3 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: Restructured the file and made it a h3.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 4 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: Corrected.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 6 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: Thanks, I've used blockquotes.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 5 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: Did an update with this in mind.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 4 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: Corrected Ma'am.

File: TMP2022/technical-writing-track/precious-tw-funke/README.md (hunk 1 above)
Comment (Vee-keme) [reply TMP2022/technical-writing-track/precious-tw-funke/README.md]: I've added a table of content

**Review by Roxie-32 — APPROVED** [review]

**Review by S-code652 — APPROVED** [review]

**Pull request merged**